index.d.ts

```diff
@@ -12,6 +14,6 @@ cryptoRandomString(10);
 //=> '2cf05d94db'
 ```
 */
-declare function cryptoRandomString(length: number): string;
+declare function cryptoRandomString(length: number, opts?: {type?: 'hex' | 'base64' | 'url-safe', characters?: string}): string;
```

Comment:
Better to create an `interface` for the options. See: https://github.com/sindresorhus/typescript-definition-style-guide

Reply:
done